tools/deployment_tool/address_generators.js

```diff
@@ -27,30 +27,30 @@ const rlp = require('rlp');
  * Suitable for genesis deployment.
  */
 class IncrementingAddressGenerator {
-    /**
+  /**
      * @param {string} [startAddress=0x0000000000000000000000000000000000010000]
      *                  Address from which we generate (by incrementing)
      *                  new addresses for contracts to deploy.
      */
-    constructor(startAddress = '0x0000000000000000000000000000000000010000') {
-        this.nextAvailableAddress = startAddress;
-    }
+  constructor(startAddress = '0x0000000000000000000000000000000000010000') {
+    this.nextAvailableAddress = startAddress;
+  }
 
-    /**
+  /**
```

Comment:
```suggestion
    /**
```
Also at other places in this file.